utils/SharedStyles.ts

```diff
@@ -0,0 +1,43 @@
+import { StyleSheet } from "react-native";
```

Comment:
Can we follow the old guidelines for colours as from the old file colors.ts which has been replaced with SharedStyles

Reply:
Instead of importing colors from colors.ts, colors are imported from SharedStyles.ts, there should be no functional difference and the colours should be up to date